.github/workflows/ci.yml

```diff
@@ -22,6 +30,16 @@ jobs:
         with:
           ghc-version: '8.10.3'
           cabal-version: '3.4'
+      - uses: actions/cache@v2
+        name: Cache cabal
+        with:
+          path: |
+            ~/.cabal
+            ~/.ghc
+            dist-newstyle
+          key: ${{ runner.os }}-cabalghc-20210330
```

Comment:
Why is this a hardcoded date?

Reply:
I'm not sure how we use a generic hash for the cabal dependencies.
So a hardcoded date may do the trick to get a unique hashkey.

Reply:
Reading online we probably need `cabal freeze`, but unsure how this works with global v1 installation commands, so perhaps we can do it later if/when we update to v2 commands.